schemes.h

```diff
@@ -26,6 +26,8 @@ struct scheme {
 #define BULCARD(s, r) \
 	ULCARD("\017\033[2m" s "\033[22m", r)//WARN: requires hack in print_hi()
 #define USCARD(c) (char*[]){c}
+#define RUSCARD(c) (char*[]){"\033[91m" c "\033[39m"}
+#define BUSCARD(c) (char*[]){c}
```

Comment:
these macros must have the same number of bytes before and including the card character. (spider's foundation prints `halfcard[2]` number of bytes)

currently, halfcard2 is 4 bytes, i.e. exactly the utf8-sequence for the char, but no colour sequence. pad buscard with `\017` 5 times before the `c` and increase halfcard2 by 5 to fix this

Reply:
on second thought, padding should be unnecessary (as it stops at the null terminator anyways). but change `halfwidth` to `{1,1,9}`

Reply:
Ah, your second comment wasn't there when I wrote my comment. ^^

Reply:
argh! my comment left by email was apparently dropped. yes, all instances of halfcard should have been `halfwidth[2]`, sorry for any confusion. i think we can do without the padding :)

Reply:
Done! :D